mesonbuild/dependencies/ui.py

```diff
@@ -493,3 +493,56 @@ def check_wxconfig(self):
                 pass
         WxDependency.wxconfig_found = False
         mlog.log('Found wx-config:', mlog.red('NO'))
+        
+class VulkanDependency(ExternalDependency):
+    def __init__(self, environment, kwargs):
+        super().__init__('vulkan', environment, None, kwargs)
+        
+        if DependencyMethods.SYSTEM in self.methods:
+            try:
+                self.vulkan_sdk = os.environ['VULKAN_SDK']
+                if not os.path.isabs(self.vulkan_sdk):
+                    raise DependencyException('VULKAN_SDK must be an absolute path.')
+                    
+                # TODO: this config might not work on some platforms, fix bugs as reported
+                # we at least have to detect other 64-bit platforms (e.g. armv8)
+                lib_name = 'vulkan'
```

Comment:
I am not sure if the loader will ever be incompatible. For opengl an explicit version is not needed as well (there is usually only one version installed), it might be handled in a similar way for vulkan.
Maybe better to wait until something like this becomes necessary and we know exactly how it should be handled?

For the moment it would only be nice to correctly report the version of the installed header correctly (since applications may e.g. use vulkan extensions introduced in 1.0.42 or something) but as stated in the TODO's i don't know a way to do this in a reliable manner.